packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -867,9 +867,10 @@ Individual .babelrc files were generated for you in src/client and src/server
           nodeRunApp = quote(Path.relative(process.cwd(), serverRun));
         }
 
+        let taskArguments = this.argv.includes("--no-ssr-sync") ? "" : taskArgs(this.argv).join(" ");
```

Comment:
please do `this.argv.filter( x => x !== "--no-ssr-sync" ).join( " " )`

Reply:
@jchip Please check now.